modules/ocf_irc/files/anope.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=A set of IRC Services
+After=network-online.target inspircd.service
+
+[Service]
+User=irc
+Group=irc
+Restart=always
+# Run ExecStartPre with root permissions to be able to create pid directory
+PermissionsStartOnly=true
+ExecStartPre=/bin/mkdir -p /var/run/anope/
```

Comment:
probably `/run/anope`? `/var/run` is a symlink to `/run` on newer versions of debian/ubuntu